assets/src/edit-story/app/api/apiProvider.js

```diff
@@ -113,6 +113,7 @@ function APIProvider({ children }) {
         context: 'edit',
         per_page: perPage,
         page: pagingNum,
+        _web_stories_envelope: 1,
```

Comment:
Why do you need to use 1 instead of true?

Reply:
Both works, just needs to be truthy. Will change to `true` though for clarity.